pkgs/tools/admin/trinsic-cli/default.nix

```diff
@@ -2,11 +2,11 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "trinsic-cli";
-  version = "1.4.0";
+  version = "1.5.0";
 
   src = fetchurl {
     url = "https://github.com/trinsic-id/sdk/releases/download/v${version}/trinsic-cli-vendor-${version}.tar.gz";
-    sha256 = "sha256-Dxmjbd1Q2JNeET22Fte7bygd+oH3ZfovRTJh5xforuw=";
+    sha256 = "028798865497919895e95da4368017b5eb342254ec9d3d592d0e14243fde744e";
```

Comment:
For the next time: we probably would want to keep the sri hash

Reply:
```suggestion
    sha256 = "sha256-Z9orGhxbu/ehyaYhY35lYWcZQWNVk+zLSoqwAZwnpLY=";
```

Reply:
Noted, thanks